packages/gatsby-telemetry/src/telemetry.js

```diff
@@ -153,6 +153,12 @@ module.exports = class AnalyticsTracker {
       enabled = true
       this.store.updateConfig(`telemetry.enabled`, enabled)
     }
+
+    // `GATSBY_TELEMETRY_DISABLED` overrides config value
+    if (this.store.disabled) {
+      enabled = false
+    }
+
```

Comment:
should this go before https://github.com/gatsbyjs/gatsby/blob/6c17305abe3ef8a8c2cfc83b9c8d730d818089d4/packages/gatsby-telemetry/src/telemetry.js#L149

Reply:
Eh, yeah 🤦‍♂ 

Wait, let me clean this up